### PR TITLE
[4.0] Custom-Select RTL

### PR DESCRIPTION
**Pull request by brianteeman**
see screenshots for changes

If we wait for an expert to fix it then it will never happen

### Before
image

### After
image

### CODE REVIEW

File: administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 .custom-select {
   max-width: $input-max-width;
+  color:red;
+  [dir=rtl] & {
+    padding: $custom-select-padding-y $custom-select-padding-x $custom-select-padding-y ($custom-select-padding-x + $custom-select-indicator-padding);
+    background-position-x: 0%;
+    background-image: url(../images/select-bg-rtl.svg);
+    }
```
Comment (Quy) [inline administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss]:
```suggestion
  }
```

File: administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 .custom-select {
   max-width: $input-max-width;
+  color:red;
```
Comment (Quy) [inline administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss]:
```suggestion
```

**Comment by wilsonge** [comment]
@infograf768 can you confirm if this fixes things for you please?

**Comment by infograf768** [comment]
@wilsonge
Will test asap, when I finish on searchtools

File: administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss [hunk 3]
```diff
@@ -2,6 +2,12 @@
 
 .custom-select {
   max-width: $input-max-width;
+  [dir=rtl] & {
```
Comment (infograf768) [inline administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss:6]: Please add empty line before `[dir=rtl] & {`

**Comment by infograf768** [comment]
Works fine after correcting the small cs above.

File: administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss (hunk 3 above)
Comment (brianteeman) [reply administrator/templates/atum/scss/vendor/bootstrap/_custom-forms.scss:6]: done as requested

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 516d0d47ea870e72b98e90e4654edc67b69a4eff<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27387">issues.joomla.org/tracker/joomla-cms/27387</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on f26ff8352c0b8f1722270235b585fb9276fa2272<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27387">issues.joomla.org/tracker/joomla-cms/27387</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27387">issues.joomla.org/tracker/joomla-cms/27387</a>.</sub>

**Comment by infograf768** [comment]
Tks.

**Comment by brianteeman** [comment]
thanks

**Pull request merged**